### PR TITLE
WF-63043 Need to update the Maps UG Content.

### PR DESCRIPTION
**Pull request by EswaranSyncfusion**
Hi @vigneshrameshsync ,

Maps UG Content is modified, So please review the content changes and let me know if you have any other concerns.

Regards,
@EswaranSyncfusion

### CODE REVIEW

File: WindowsForms/Map/Customizing-Map-appearance.md [hunk 1]
```diff
@@ -9,6 +9,88 @@ documentation: ug
...
+         }
+} 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Screenshot:
+
+![Equal Color Mapping in Windows Forms Map](Customizing-Map-appearance_images/Customizing-Map-appearance_img2.png)
```
Comment (vigneshrameshsync) [inline WindowsForms/Map/Customizing-Map-appearance.md:90]: Provide proper name for image like "customizing-map-appearance-with-equal-color-mapping"

Comment (vigneshrameshsync) [inline WindowsForms/Map/Customizing-Map-appearance.md:90]: If the LabelIntersection option is there, then set it as trim or hide. It is very odd to see the intersection labels in output image.

**Review by vigneshrameshsync — CHANGES_REQUESTED** [review]

File: WindowsForms/Map/Customizing-Map-appearance.md (hunk 1 above)
Comment (EswaranSyncfusion) [reply WindowsForms/Map/Customizing-Map-appearance.md]: Proper image name is provided

Comment (EswaranSyncfusion) [reply WindowsForms/Map/Customizing-Map-appearance.md]: LabelIntersection option is not there so i hide the value text.

**Review by vigneshrameshsync — APPROVED** [review]
The changes are fine.

**Review by sheiksyedm — APPROVED** [review]

**Pull request merged**